ui/selectors/selectors.js

```diff
@@ -2911,6 +2911,13 @@ export function getMetaMetricsDataDeletionStatus(state) {
   return state.metamask.metaMetricsDataDeletionStatus;
 }
 
+export function getRemoteFeatureFlagsByName(state, featureFlagName) {
```

Comment:
I think `state.metamask.remoteFeatureFlags?.featureFlagName` should be `state.metamask.remoteFeatureFlags?.[featureFlagName]`

Reply:
and this should return `state.metamask.remoteFeatureFlags?.[featureFlagName]`

Reply:
so that `getRemoteFeatureFlagsByName` returns just a single feature flag, the one named by `featureFlagName`

Reply:
Although, it might be simplest to just have a `getRemoteFeatureFlags` function that always returns `state.metamask.remoteFeatureFlags`, and then we can just go `remoteFeatureFlags[NAME]` in the location that the feature flags are needed. This also has the benefit of the feature flag name being in the same code location as the use, making it clear what specific feature flag is in use.

Reply:
Makes sense ! reverted 89fe7aced2e9557d1841598bead82c5ac897cfb9

Reply:
Addressed in 89fe7aced2e9557d1841598bead82c5ac897cfb9